tests/test_model_saving.py

```diff
@@ -232,5 +232,35 @@ def test_loading_weights_by_name_2():
     assert_allclose(np.zeros_like(jessica[1]), jessica[1])  # biases init to 0
 
 
+# a function to be called from the Lambda layer
+def square_fn(x):
+    return x + x
```

Comment:
Technically not the square function... also, it may be better to define it inside the test function. It shouldn't change anything to what we are testing here.